docs/com1DFAAlgorithm.rst

```diff
@@ -81,12 +82,50 @@ The mass per particle determination method can be chosen between:
            particles per cell. Finally, using the MPPKR method, the number of particles per cell is independent from
            both cell size and release thickness (``nPPK`` particles per kernel radius is the target value).
 
-The number of particles placed in each release cell is computed according to the ``massPerPart`` or ``nPPK`` depending
+The number of particles placed in each release cell ``nPartPerCell`` is computed according to the ``massPerPart`` or ``nPPK`` depending
 on the ``massPerParticleDeterminationMethod`` chosen and the area and/or release thickness of the cell.
 The number should be an integer meaning that the float is rounded up or down with a probability corresponding to the
 decimal part (i.e. 5.7 will be rounded to 6 with a probability of 0.7 and 5 with a probability of 0.3).
-This ensures a better match with the desired ``massPerPart`` value. Particles are then placed randomly within the
-mesh cell.
+This ensures a better match with the desired ``massPerPart`` value.
+
+There are then different ways to place the particles in the cells. This is decided by the
+``initPartDistType`` parameter in the configuration file:
+
+    - ``random`` initialization:
+      The ``nPartPerCell`` particles are placed randomly within each release cell.
+      This initialization method allows steps of ones in the choice of ``nPartPerCell`` which enables to
+      avoid large jumps in the mass of the particles between one release cell to another.
+      Due to the random initialization process, some particles cluster can appear. For operational applications
+      this does not seem to have a significant impact since the particles will redistribute in the first
+      few time steps due to the pressure gradient. For some specific research application,
```

Comment:
is the dam break test an example for this? so when we look at the early stages of the flow?